components/planview_leankit/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@pipedream/planview_leankit",
+  "version": "0.0.1",
+  "description": "Pipedream Pagerduty Components",
```

Comment:
I think the app name here might be a leftover from another package and needs to be updated